extensions/cornerstone/src/components/StudyMeasurements.tsx

```diff
@@ -26,13 +28,19 @@ export const groupByStudy = (items, grouping, childProps) => {
     const studyUID = getItemStudyInstanceUID(item);
     if (!groups.has(studyUID)) {
       const items = [];
+      const filter = MeasurementFilters.filterAnd(
```

Comment:
The study measurements filter needs to define the filter object so that saving and other operations apply on a per study basis.